### PR TITLE
[Fluent] Map

### PR DESCRIPTION
**Pull request by kikostadinov**
Related to: https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/map/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,134 @@
+@use "_variables.scss" as *;
+
+@mixin kendo-map--layout() {
+
+    // Layout
+    .k-map {
+        height: var( --$kendo-map-height, #{$kendo-map-height} );
```
Comment (Juveniel) [inline packages/fluent/scss/map/_layout.scss:7]:
```suggestion
height: var( --kendo-map-height, #{$kendo-map-height} );
```

File: packages/fluent/scss/map/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+/// Map attribution horizontal padding.
+/// @group map
+$kendo-map-attribution-padding-x: $kendo-padding-x-sm !default;
+/// Map attribution vertical padding.
+/// @group map
+$kendo-map-attribution-padding-y: $kendo-padding-y-sm !default;
+/// Map attribution font size.
+/// @group map
+$kendo-map-attribution-font-size: calc($kendo-map-font-size * .75) !default;
```
Comment (Juveniel) [inline packages/fluent/scss/map/_variables.scss:79]: spacing

File: packages/utils/scss/layout/_position.scss [hunk 3]
```diff
@@ -58,3 +58,153 @@ $kendo-utils-inset: map-get( $kendo-utils, "inset" ) !default;
...
+.k-pos-top-start {
+    top: 0;
+    left: 0;
+
+    .k-rtl &,
+    &[dir="rtl"],
+    [dir="rtl"] & {
+        left: auto;
+        right: 0;
+    }
```
Comment (Juveniel) [inline packages/utils/scss/layout/_position.scss:97]: you can remove the rtl parts:

```suggestion
.#{$kendo-prefix}top-start,
.#{$kendo-prefix}pos-top-start {
    inset-block-start: 0;
    inset-inline-start: 0;
}
```

File: packages/utils/scss/layout/_position.scss [hunk 4]
```diff
@@ -58,3 +58,153 @@ $kendo-utils-inset: map-get( $kendo-utils, "inset" ) !default;
...
+/// @description This is equivalent to `right: 0;`.
+/// @group position
+
+/// @name k-pos-bottom
+/// @description This is equivalent to `bottom: 0;`.
+/// @group position
+
+/// @name k-pos-left
+/// @description This is equivalent to `left: 0;`.
+/// @group position
```
Comment (Juveniel) [inline packages/utils/scss/layout/_position.scss:77]: no needs of docs for these, as we will deprecate them in favor of the above-generated map at some point after the release, thus please remove these

File: packages/utils/scss/layout/_position.scss [hunk 5]
```diff
@@ -58,3 +58,153 @@ $kendo-utils-inset: map-get( $kendo-utils, "inset" ) !default;
...
+
+@each $side in (top, right, bottom, left) {
+    .k-#{$side},
+    .k-pos-#{$side} { #{$side}: 0; } // sass-lint:disable-line brace-style
+}
+
+/// @name k-pos-top-start
+/// @description This is equivalent to `top: 0; left: 0;`.
+/// @group position
```
Comment (Juveniel) [inline packages/utils/scss/layout/_position.scss:86]: same for these

File: packages/utils/scss/layout/_position.scss [hunk 6]
```diff
@@ -58,3 +58,153 @@ $kendo-utils-inset: map-get( $kendo-utils, "inset" ) !default;
...
+/// @description This is equivalent to `bottom: 0;`.
+/// @group position
+
+/// @name k-pos-left
+/// @description This is equivalent to `left: 0;`.
+/// @group position
+
+@each $side in (top, right, bottom, left) {
+    .k-#{$side},
+    .k-pos-#{$side} { #{$side}: 0; } // sass-lint:disable-line brace-style
```
Comment (Juveniel) [inline packages/utils/scss/layout/_position.scss:81]: use .#{$kendo-prefix}

**Review by Juveniel — CHANGES_REQUESTED** [review]

File: packages/fluent/scss/map/_variables.scss [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+$kendo-map-navigator-margin-x: map-get( $kendo-spacing, 4 ) !default;
+/// Map navigator vertical margin.
+/// @group map
+$kendo-map-navigator-margin-y: map-get( $kendo-spacing, 4 ) !default;
+/// Map navigator padding.
+/// @group map
+$kendo-map-navigator-padding: map-get( $kendo-spacing, thin ) !default;
+/// Map navigator width.
+/// @group map
+$kendo-map-navigator-width: calc( #{$kendo-icon-size * 3} + #{$kendo-map-navigator-padding * 2} ) !default;
```
Comment (TeyaVes) [inline packages/fluent/scss/map/_variables.scss:44]: use the CSS var for the icon-size exposed in the root

File: packages/fluent/scss/map/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,89 @@
+@use "../core/color-system" as *;
+@use "../core/_variables" as *;
+@use "../icon/_variables.scss" as *;
```
Comment (TeyaVes) [inline packages/fluent/scss/map/_variables.scss:3]: no need for this if the CSS vars for the icons are used

File: packages/fluent/scss/map/images/markers.scss [hunk 9]
```diff
@@ -0,0 +1,3 @@
...
...UmqUUkhEYYn9PamgbROKTEERB1HNP0pMvJUVhZQhCEEQPQiAIkiQIQShEZmg99SdLgiRJpbGmGIu/aZ2zzJXrvWf2zty7PgTzwQd3Z86Z796z58wMJ+PruXxQwCHkGeRpZDbyqBh/j1xA9iN7kJ+8FsrwECSRu8hSUMMosgnZLTMIScaLkUPIpxpiIGyfCN9iVcEo8h2yCvyjSqwR9RIkgz5kHgRHnlhrg2im7blEhCPCeW89fBJyKuohsv8IRPaV46uGYfnLhyTn3/bC4seXnFtErFmBHHcmzSCy2umRWVgC+ZfvQ1bZ8ZSfsxQbgl+PrsCfqXFu+jWyxh7Ss5xY1sFKKLzd5ymWtEWbpC36MKgWGuuCzdyXFTR2QGhbrvKfRrbkQ74Mmi3BKJf6FEYdMbso+UpKJkpJU+dKkPIaNoyrM5Mw23Edlj6/WQ957qVbEM4rcoWX1lgcGXQuURfi6o2y0YmVH2Mw2XgC5od6ksJEeqaxlW+joLIGaZFgGVcCTsx13YG1xKxrnMZoTmUN0iLB7c7R8M49LktJnSWxwMxxa5AWu5f+XV6EzQIJznHJwSWSDNlM+Fanv3OmcyQY43YNVyzOX2XLhMZoTvEviJHgsHOU9kYntuwthaLWV5BTWQ/hHbuSpGcaozmVNUiL9lIqfNfr7L7Rq7SlyfbVnzfZsjhFXzggTuoNoI14bf63thj5kK/kNvDCytIm5yzt+vF7F9jak4qhbbz1ouzEaLJv3t3iCNkYGtzCpq7Vrm9lKcNo2TIJJ/Kk23keHhB3kQLPAxhJUDiACXEkJcMYd2uzrhiRNNX5MrJW5Al7pxkQBjNpEJtxislubWRwjKtPDQyLNQZU76UTIu4NPsQahO+EzkXYwgNu60uBmPAB3Zu3HS0agp62KoJdsvAwf0NXOgQptdsU7NqEbWBBQrso4FQl0K56AKsggXzokVyJdAqmWjThlZl+BeOSsD32CLdvQSsxVu1XF2Sr7iVKB5T6nbbfnYol41vQWdwtus6ZPgRp+3pme950QV9fpito+jSmT2P6NKZP479PE9Lp06jC6tMwUO/T6CJwn0YXgfs0uvj/+jS6CNyn0UXgPo0uZH0aEnzueruRwUBfSb5Mr5TQb/o0pk9j+jSmT2P6NKZPY/o0pk9j+jQ+8E+AAQDpbNHEyW7DTAAAAABJRU5ErkJggg==");
```
Comment (TeyaVes) [inline packages/fluent/scss/map/images/markers.scss:2]: remove this image as we don't seem to use it anymore

**Review by TeyaVes — CHANGES_REQUESTED** [review]

**Review by Juveniel — APPROVED** [review]

**Pull request merged**